### ISSUE TITLE
Deploy command ignores --tag argument

### ISSUE THREAD

**Issue by kfrance** [balena-io/balena-cli#1018]
- Using cli version 9.1.0
- Using intel x86 archlinux system that is up to date

The tag argument is ignored when using deploy. This seems to be related to issue #825 where the tag argument is also ignored for the build command.

**Comment by kfrance** [comment]
I tried pushing with projectName but I don't see any tags on the release, or anything else related to the project name I gave it.

**Comment by thgreasi** [comment]
@kfrance can I get some context on what you are trying to achieve and the complete command you are using for that?
The `--projectName` parameter is only used in the deploy command when you are also using the `--build` parameter and it affects the name of the build image.
You can check the name of the resulting image name locally by running `docker images`, which will also have a suffix with the name of the service that this image is for (for single container apps that would be `myprojectname_main`).

**Comment by pdcastro** [comment]
Internal discussion thread: https://www.flowdock.com/app/rulemotion/i-cli/threads/S2M8AnPbIbbH6qKQHUOEb9lYDYb

**Comment by pdcastro** [comment]
Fixed in CLI v12.48.11. The `--tag` flag specifies only the `:tag` portion of the image name. For example:
```
$ balena deploy myFleet --build -s . --projectName dronecontrol --tag v2.2.4
```
Images created (e.g. as listed by `docker images` on the workstation):
```
dronecontrol_service1:v2.2.4
dronecontrol_service2:v2.2.4
dronecontrol_service3:v2.2.4
```
The default if `--tag` is not specified is `:latest`.

Help output:

```
$ balena help deploy
...
OPTIONS
...
-n, --projectName <projectName>  Name prefix for locally built images. This is the 'projectName' portion
                                 in 'projectName_serviceName:tag'. The default is the directory name.

-t, --tag <tag>                  Tag locally built Docker images. This is the 'tag' portion
                                 in 'projectName_serviceName:tag'. The default is 'latest'.
```
